### PR TITLE
trivy: edit page

### PR DESCRIPTION
**Pull request by fazlearefin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/trivy.md [hunk 1]
```diff
@@ -1,20 +1,36 @@
...
 > Scanner for vulnerabilities in container images, file systems, and Git repositories, as well as for configuration issues.
-> More information: <https://github.com/aquasecurity/trivy>.
+> More information: <https://aquasecurity.github.io/trivy>.
 
-- Scan an image:
+- Scan a Docker image for vulnerabilities and exposed secrets:
 
 `trivy image {{image:tag}}`
 
+- Scan a Docker image filtering by severities:
```
Comment (gutjuri) [inline pages/common/trivy.md:10]:
```suggestion
- Scan a Docker image filtering the output by severity:
```

File: pages/common/trivy.md [hunk 2]
```diff
@@ -1,20 +1,36 @@
...
-- Scan a directory for misconfigurations:
+- Scan a IaC (Terraform, CloudFormation, ARM, Helm and Dockerfile) directory for misconfigurations:
 
 `trivy config {{path/to/iac_directory}}`
 
+- Scan a local or remote git repository for vulnerabilities:
+
+`trivy repo {{path/to/local_repository_directory|remote_repository_URL}}`
+
+- Scan a git repository upto a specific commit hash:
```
Comment (gutjuri) [inline pages/common/trivy.md:30]:
```suggestion
- Scan a git repository up to a specific commit hash:
```

So it scans all commits from the initial commit to the one specified?

File: pages/common/trivy.md [hunk 3]
```diff
@@ -1,20 +1,36 @@
 # trivy
 
 > Scanner for vulnerabilities in container images, file systems, and Git repositories, as well as for configuration issues.
-> More information: <https://github.com/aquasecurity/trivy>.
+> More information: <https://aquasecurity.github.io/trivy>.
 
-- Scan an image:
+- Scan a Docker image for vulnerabilities and exposed secrets:
```
Comment (gutjuri) [inline pages/common/trivy.md:6]:
```suggestion
- Scan an image for vulnerabilities and exposed secrets:
```

Only docker images or also other kinds of images?

File: pages/common/trivy.md [hunk 4]
```diff
@@ -1,20 +1,36 @@
...
 
 `trivy image {{image:tag}}`
 
+- Scan a Docker image filtering by severities:
+
+`trivy image --severity {{HIGH,CRITICAL}} {{alpine:3.15}}`
+
+- Scan a Docker image ignoring any unfixed/unpatched vulnerabilities:
```
Comment (gutjuri) [inline pages/common/trivy.md:14]:
```suggestion
- Scan an image ignoring any unfixed/unpatched vulnerabilities:
```

see above

**Review by gutjuri — COMMENTED** [review]
Thanks, I have some comments

File: pages/common/trivy.md (hunk 3 above)
Comment (fazlearefin) [reply pages/common/trivy.md:6]: > Only docker images or also other kinds of images?

Docker image only when subcommand is `image`. Reference here: https://aquasecurity.github.io/trivy/v0.49/docs/target/container_image/

File: pages/common/trivy.md (hunk 4 above)
Comment (fazlearefin) [reply pages/common/trivy.md:14]: As mentioned above `image` means docker image here. Resolving conversation.

**Review by gutjuri — APPROVED** [review]

File: pages/common/trivy.md [hunk 5]
```diff
@@ -1,20 +1,36 @@
...
-- Scan a directory for misconfigurations:
+- Scan a IaC (Terraform, CloudFormation, ARM, Helm and Dockerfile) directory for misconfigurations:
 
 `trivy config {{path/to/iac_directory}}`
 
+- Scan a local or remote git repository for vulnerabilities:
+
+`trivy repo {{path/to/local_repository_directory|remote_repository_URL}}`
+
+- Scan a git repository up to a specific commit hash:
```
Comment (kbdharun) [inline pages/common/trivy.md:30]:
```suggestion
- Scan a local or remote Git repository for vulnerabilities:

`trivy repo {{path/to/local_repository_directory|remote_repository_URL}}`

- Scan a Git repository up to a specific commit hash:
```

**Review by vitorhcl — APPROVED** [review]

**Pull request merged**